nvs/parser.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Nikkei Value Search
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+  if (/^\/macro\/[a-z]+$/i.test(path)) {
+    // https://valuesearch.nikkei.com/macro/statistics?corpIndustyKbn=5&keyword=work
+    result.rtype    = 'TOOL';
+    result.mime     = 'HTML';
+
+  } else if (/^\/popup$/i.test(path)) {
+    // https://valuesearch.nikkei.com/popup?keyBody=DISNWS140120220815520127&transitionId=7dfddbd2-d0fb-47f9-bba7-1c8a11e4ff63&tldTransitionId=&corpIndustyKbn=6&pathname=/searchlist/news&mediaFeeType=0&keyBodyFare=0&articleId=DISNWS140120220815520127&pdfViewer=true&keywords=work
+    result.rtype    = 'ISSUE';
+    result.mime     = 'HTML';
+    result.unitid   = param.keyBody;
```

Comment:
`/popup` looks rather generic, maybe we could use `articleId` to narrow the condition ?

```suggestion
  } else if (/^\/popup$/i.test(path) && param.articleId) {
    // https://valuesearch.nikkei.com/popup?keyBody=DISNWS140120220815520127&transitionId=7dfddbd2-d0fb-47f9-bba7-1c8a11e4ff63&tldTransitionId=&corpIndustyKbn=6&pathname=/searchlist/news&mediaFeeType=0&keyBodyFare=0&articleId=DISNWS140120220815520127&pdfViewer=true&keywords=work
    result.rtype    = 'ISSUE';
    result.mime     = 'HTML';
    result.unitid   = param.articleId;
```

Reply:
made this change